src/AppBundle/Resources/views/Email/report-submission-confirm.html.twig

```diff
@@ -4,18 +4,26 @@
 {% trans_default_domain translationDomain %}
 
 {% set transOptions = {
-    '%deputyFirstName%': deputyFirstName,
+    '%fullDeputyName%': fullDeputyName,
     '%startDate%': submittedReport.startDate | date("d/m/Y"),
     '%endDate%': submittedReport.endDate | date("d/m/Y"),
     '%newStartDate%': newReport.startDate | date("d/m/Y"),
     '%newEndDate%': newReport.endDate | date("d/m/Y"),
-    '%homepageUrl%': homepageUrl
+    '%homepageUrl%': homepageUrl,
+    '%fullClientName%': fullClientName,
+    '%caseNumber%': caseNumber
 } %}
 
 {% block emailHTMLHeaderTitle %}{{ ('default.htmlHeaderTitle') | trans }}{% endblock %}
 
 {% block content %}
 
+    {% if recipientRole != 'default' %}
```

Comment:
Professionals and PAs only?

Reply:
Yeah - confirmed in the ticket. From what I could see in `MailFactory::getRecipientRole()`, anything other than PA or Professional will be set to `default` so hopefully this should be enough?

Reply:
Yes, checked and confirmed, default it seems to be.

Reply:
Confirmed